client/my-sites/themes/current-theme/index.jsx

```diff
@@ -65,28 +67,47 @@ class CurrentTheme extends Component {
 				<QueryActiveTheme siteId={ siteId } />
 				{ currentThemeId && <QueryCanonicalTheme themeId={ currentThemeId } siteId={ siteId } /> }
 				<div className="current-theme__current">
-					{ showScreenshotPlaceholder && <div className="current-theme__img-placeholder" /> }
-					{ showScreenshot && (
-						<img src={ currentTheme.screenshot + '?w=150' } className="current-theme__img" alt="" />
-					) }
-					<span className="current-theme__label">{ translate( 'Current Theme' ) }</span>
-					<span className="current-theme__name">{ text }</span>
-				</div>
-				<div
-					className={ classNames( 'current-theme__actions', {
-						'two-buttons': Object.keys( options ).length === 2,
-					} ) }
-				>
-					{ map( options, ( option, name ) => (
-						<CurrentThemeButton
-							name={ name }
-							key={ name }
-							label={ option.label }
-							icon={ option.icon }
-							href={ currentThemeId && option.getUrl( currentThemeId ) }
-							onClick={ this.trackClick }
-						/>
-					) ) }
+					<div className="current-theme__details">
+						{ showScreenshotPlaceholder && <div className="current-theme__img-placeholder" /> }
+						{ showScreenshot && (
+							<img
+								src={ currentTheme.screenshot + '?w=150' }
```

Comment:
We might need a larger size if the small screen allows the screenshot to go full width.

Reply:
Right, looks like it can be full width up to 660px.